paper/paper.md

```diff
@@ -11,49 +14,57 @@ authors:
     orcid: 0000-0002-2415-2186
     affiliation: 1
   - name: Lukas Fehring
+    orcid: 0000-0001-8057-4650
     affiliation: 1
   - name: Lukas Gehring
     affiliation: 1
   - name: Helena Graf
+    orcid: 0000-0001-9447-0609
     affiliation: 1
   - name: Jonas Hanselle
+    orcid: 0000-0002-1231-4985
     affiliation: 1
   - name: Marcel Wever
+    orcid: 0000-0001-9782-6818
     affiliation: 2
-  - name: Felix Mohr
+  - name: Felix Mohr 
+    orcid: 0000-0002-9293-2424
     affiliation: 3
 affiliations:
  - name: Department of Computer Science, Paderborn University, Germany
    index: 1
- - name: Institut of Informatics, University of Munich, Germany
+ - name: MCML, Institut for Informatics, LMU Munich, Germany
    index: 2
  - name: Universidad de La Sabana, Chia, Cundinamarca, Colombia
    index: 3
-date: 16 August 2022
+date: 4 November 2022
 bibliography: paper.bib
 
 ---
 
 # Summary
 
-The `PyExperimenter` is a tool for the automatic execution of, e.g. machine learning (ML) experiments and capturing corresponding results in a unified manner in a database. It supports both sqlite or mysql backends. The experiments to conduct can be defined via a configuration file or in custom way through code. Based on that, a table with initial properties, e.g. different seeds for an ML algorithm, is filled. During execution, a custom defined function requested to compute the results for a single row of the table. Those results, e.g. performances, can be added to the table at the end of the execution. Errors occurring during the execution are logged in the database. Afterwards, experiment evaluation tables can be easily extracted, e.g. averaging over different seeds. 
+`PyExperimenter` is a tool to facilitate the setup, documentation, execution, and subsequent evaluation of results from an empirical study of algorithms and in particular is designed to reduce the involved manual effort significantly. It was designed to be used by researchers in the field of artificial intelligence, but is not limited to those.
```

Comment:
Duplicate 'designed', I suggest starting the second sentence with. e.g. 'It is intended to be used ...'